src/features/collapsible-sidebar/CollapsibleSidebarLogo.js

```diff
@@ -72,13 +72,15 @@ type Props = {
     /** Controls whether or not the sidebar is expanded on the page */
     expanded: boolean,
 
+    hideBoxLogo: boolean,
```

Comment:
can we do `isLogoVisible` and make it optional? we'd need to default it to `true`

Reply:
Yes, I changed the prop name and set it to true by default.